File: python/Tiket.py
class Tiket:

    #constructor
    def __init__(self, idTiket:str, namaFilm:str, studio:str, jamTayang:str, nomorKursi:str, harga:int):
        self.__idTiket = str(idTiket)
        self.__namaFilm = str(namaFilm)
        self.__studio = str(studio)
        self.__jamTayang = str(jamTayang)
        self.__nomorKursi = str(nomorKursi)
        self.__harga = int(harga)

    #getter
    def getIdTiket(self) -> str:
        return self.__idTiket

    def getNamaFilm(self) -> str:
        return self.__namaFilm

    def getStudio(self) -> str:
        return self.__studio

    def getJamTayang(self) -> str:
        return self.__jamTayang

    def getNomorKursi(self) -> str:
        return self.__nomorKursi

    def getHarga(self) -> int:
        return self.__harga

    #setter
    def setIdTiket(self, idTiket:str) -> None:
        self.__idTiket = str(idTiket)

    def setNamaFilm(self, namaFilm:str) -> None:
        self.__namaFilm = str(namaFilm)

    def setStudio(self, studio:str) -> None:
        self.__studio = str(studio)

    def setJamTayang(self, jamTayang:str) -> None:
        self.__jamTayang = str(jamTayang)

    def setNomorKursi(self, nomorKursi:str) -> None:
        self.__nomorKursi = str(nomorKursi)

    def setHarga(self, harga:int) -> None:
        self.__harga = int(harga)

File: java/Tiket.java
public class Tiket {

    // constructor
    public Tiket(String idTiket, String namaFilm, String studio,
                 String jamTayang, String nomorKursi, int harga) {

        this.idTiket = String.valueOf(idTiket);
        this.namaFilm = String.valueOf(namaFilm);
        this.studio = String.valueOf(studio);
        this.jamTayang = String.valueOf(jamTayang);
        this.nomorKursi = String.valueOf(nomorKursi);
        this.harga = harga;
    }

    // getter
    public String getIdTiket() {
        return idTiket;
    }

    public String getNamaFilm() {
        return namaFilm;
    }

    public String getStudio() {
        return studio;
    }

    public String getJamTayang() {
        return jamTayang;
    }

    public String getNomorKursi() {
        return nomorKursi;
    }

    public int getHarga() {
        return harga;
    }

    // setter
    public void setIdTiket(String idTiket) {
        this.idTiket = String.valueOf(idTiket);
    }

    public void setNamaFilm(String namaFilm) {
        this.namaFilm = String.valueOf(namaFilm);
    }

    public void setStudio(String studio) {
        this.studio = String.valueOf(studio);
    }

    public void setJamTayang(String jamTayang) {
        this.jamTayang = String.valueOf(jamTayang);
    }

    public void setNomorKursi(String nomorKursi) {
        this.nomorKursi = String.valueOf(nomorKursi);
    }

    public void setHarga(int harga) {
        this.harga = harga;
    }

    // attribute
    private String idTiket;
    private String namaFilm;
    private String studio;
    private String jamTayang;
    private String nomorKursi;
    private int harga;
}